### DISCUSSION TITLE
I don’t understand how to get Github Education

### DISCUSSION THREAD

**Discussion by ravilovrm** [community/community#134402] [GitHub Education]
### Hi everyone,

Hi, everyone

I tried many times to send them this file (photographed, I took the photo both on my mobile phone and on my computer’s webcam) but was always refused. I don't use VPN. Moreover, when I select my school on the first page of submission (it is there), then upload the documents, I immediately receive a refusal (instantly) by email. When I enter the name of my school (School 292 of St. Petersburg), they consider it for several days, but still reject it. What should I do?

I attach to them the original student ID in Russian and a printed copy in English
![Безымянный72](https://github.com/user-attachments/assets/f65a0b67-95d1-40ff-b817-666ffebafad5)

**Comment by akshit2941** [comment]
try to set up two factor auth and adding billing details to your profile
